tests/specs/tests.scss

```diff
@@ -1,7 +1,17 @@
 @import "true";
 
-// Import all scss files
-@import "ivory";
+// Import ivory
+@import "ivory.tests";
+
+// Set all prefixes to true for tests.
+// - - - - - - - - - - - - - - - - - - - - - - - - -
+@include config-set("prefix.webkit", true);
+@include config-set("prefix.mozilla", true);
+@include config-set("prefix.microsoft", true);
+@include config-set("prefix.opera", true);
+@include config-set("prefix.spec", true);
 
 // Import scss test files
-@import "scss/helpers/direction.test";
+@import "scss/components/css3/animation.test";
+@import "scss/components/css3/_appearance.test";
```

Comment:
Imported partial `scss/components/css3/_appearance.test` should be written as `scss/components/css3/appearance.test`